### DISCUSSION TITLE
什么时候关闭V3的官网

### DISCUSSION THREAD

**Discussion by baiban114** [antvis/G6#6396] [General]
https://antv-2018.alipay.com/zh-cn/g6/3.x/index
该网站在bing搜索中被排到了第一位（顺便一提搜不到现在的官网）在里面点进语雀之后甚至连使用手册也找不到。倒是在语雀中还有G的使用文档，我这一路找过去也没提示说G6.x和G6是两个项目, 结果按照G的手册使用，浪费了许多时间

**Comment by Aarebecca** [comment]
你好，我们会尽快解决该问题，非常感谢反馈
